common/src/configuration/global.rs

```diff
@@ -77,9 +77,7 @@ pub struct GlobalConfig {
     pub base_node_tor_identity_file: PathBuf,
     pub wallet_db_file: PathBuf,
     pub console_wallet_db_file: PathBuf,
-    pub wallet_identity_file: PathBuf,
     pub console_wallet_identity_file: PathBuf,
-    pub wallet_tor_identity_file: PathBuf,
     pub console_wallet_tor_identity_file: PathBuf,
```

Comment:
We should probably look at removing these as well. The wallet is not supposed to use them anymore. Its saves this data inside of the database.

Reply:
Agreed, they should be removed, but I think it for another PR, has to do with the console wallet.